docker-compose.yml

```
@@ -141,22 +144,26 @@ services:
       context: ./openshift/docker/backend
       dockerfile: Dockerfile.dev
     hostname: backend
+    env_file:
+    - path: ./env.test
+      required: false
     environment:
+      ENVIRONMENT: "${ENVIRONMENT}"
       APP_CONTEXT_ROOT: gwells
       CSRF_COOKIE_SECURE: "False"
       CUSTOM_GDAL_GEOS: "False"
       DATABASE_NAME: gwells
-      DATABASE_USER: gwells
-      DATABASE_PASSWORD: test1
+      DATABASE_USER: "${DATABASE_USER}"
+      DATABASE_PASSWORD: "${DATABASE_PASSWORD}"
       DATABASE_SERVICE_NAME: gwells
       DJANGO_ADMIN_URL: admin
       DJANGO_DEBUG: "True"
       DJANGO_SECRET_KEY: secret
       ENABLE_ADDITIONAL_DOCUMENTS: "True"
       ENABLE_AQUIFERS_SEARCH: "True"
       ENABLE_GOOGLE_ANALYTICS: "False"
-      GWELLS_SERVICE_HOST: db
-      GWELLS_SERVICE_PORT: 5432
+      GWELLS_SERVICE_HOST: "${GWELLS_SERVICE_HOST}"
+      GWELLS_SERVICE_PORT: "${GWELLS_SERVICE_PORT}"
```

```suggestion
      GWELLS_SERVICE_HOST: "${GWELLS_SERVICE_HOST:-db}"
      GWELLS_SERVICE_PORT: "${GWELLS_SERVICE_PORT:-5432}"
```
You should be able to add default arguments to your env variables by doing something like this.
Tightens the docker compose, and removes the need for that default .env that risks commits